GoogleDataTransport/GDTCORLibrary/Private/GDTCORMetricsMetadata.h

```diff
@@ -28,7 +28,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, copy, readonly) NSDate *collectionStartDate;
 
 /// The dropped event counter associated with the metrics.
-@property(nonatomic, copy, readonly, nullable) GDTCOREventMetricsCounter *droppedEventCounter;
```

Comment:
This is no longer `nullable`. The initializer below was updated in a previous PR and I realized the discrepancy in this PR